module/Activity/src/Model/Activity.php

```diff
@@ -469,7 +470,7 @@ public function setRequireGEFLITST($requireGEFLITST)
     }
 
     /**
-     * @return ArrayCollection
```

Comment:
These are `ArrayCollection`s and `Collection` is an interface (which is implemented by `ArrayCollection`), I don't think we should change them (applies to most, if not all, changes in this commit).

Reply:
If we don't use any other functionality that is not present in ArrayCollection we should use Collection. Interfaces change less often and it thus makes our code more maintainable :)